code-challenges.test.js

```diff
@@ -45,5 +156,37 @@ const testArray1 = [3, 7, 10, 5, 4, 3, 3]
 const testArray2 = [7, 8, 2, 3, 1, 5, 4]
 // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
 
+// describe("noDuplicates", () => {
+//     const expected = [3, 7, 10, 5, 4, 8, 2, 1]
+//   it("takes in two arrays as arguments and returns one array with no duplicate values.", () => {
+//     expect(noDuplicates(testArray1,testArray2)).toEqual(noDuplicates(expected))
+//   })
+// })
+// ReferenceError: noDuplicates is not defined
+// PASS
+
 
 // b) Create the function that makes the test pass.
+
+// Pseudo code:
+// Declare a function called noDuplicates
+// Pass parameters: array1, array2
+// Delcare a varibale called comboArray. 
+// Use .concat to combine arrays. Give comboArray the value -- array1.concat(array2)
+// The Set method holds any value and creates a collection of instances of the values.  The values being passed in can only be used once. Therefore preventing duplicate values.
+// Declare a varibale called newArray and give it the value [...new Set(comboArray)]
+// Return newArray
+
+const noDuplicates = (array1,array2) => {
+    let comboArray = array1.concat(array2)
+    let newArray =[...new Set(comboArray)]
+    return newArray
+
+}
+// console.log(noDuplicates(testArray1, testArray2)) // [3, 7, 10, 5, 4, 8, 2, 1]
+
+
+
+
```

Comment:
Beautiful!  Great explanation in your pseudocode.  Love that you found and used Set().  You're killing it!